### ISSUE TITLE
Dockerfile-minimal

### ISSUE THREAD

**Issue by grantmacken** [eXist-db/docker-existdb#9]
Dockerfile-minimal , this won't build, I think it depended on build.sh.
My suggestion remove.
If we want targeted minimal build, add a phased target  in the multi staged Dockerfile

```
docker build --target='min' --tag ${DOCKER_IMAGE}:min-v3.2.1
```

**Comment by adamretter** [comment]
I think this might be some old code? I think @duncdrum mentioned that there is only one build now and that it is the minimal build. @duncdrum ?

**Comment by duncdrum** [comment]
already fixed in the fixed-link branch see #6 #7
